docs/form-datasets.rst

```diff
@@ -56,19 +71,21 @@ These files may also have any number of additional columns used in :ref:`choice
   ezeke_amasiri, Ezeke amasiri, afikpo_n
   poperi_amasiri, Poperi amasiri, afikpo_n
 
+.. _selects-from-entity-lists:
+
+Building selects from Entity Lists
+-----------------------------------
+
+Entity Lists can be used as choice lists for select questions using ``select_one_from_file`` or ``select_multiple_from_file``. All Entities must have a ``name`` and ``label`` which are used by default as the value and label for choices. If you would like to use other properties as values or labels, you can specify those in the :ref:`parameters column <customizing-label-and-value>` of your XLSForm. Additional properties can be used in choice filters or other expressions, just like the columns in a CSV.
+
 .. _selects-from-geojson:
 
 Building selects from GeoJSON files
 ------------------------------------
 
-*New in* `ODK Collect v2022.2.0 <https://github.com/getodk/collect/releases/tag/v2022.2.0>`_, `ODK Central v1.4.0 <https://forum.getodk.org/t/odk-central-v1-4/36886>`_
-
-.. warning::
-  GeoJSON attachments are not yet available in web forms (Enketo).
+*New in* `ODK Collect v2022.2.0 <https://github.com/getodk/collect/releases/tag/v2022.2.0>`_, `ODK Central v1.4.0 <https://forum.getodk.org/t/odk-central-v1-4/36886>`_; Polygons and lines in Collect v2023.1.0
```

Comment:
I find the polygons and lines addition hard to grok. 

I'd just mention the newest version so you don't have to mention the polygons.
> *New in* `ODK Collect v2023.1.0 <https://github.com/getodk/collect/releases/tag/v2023.1.0>`_, `ODK Central v1.4.0 <https://forum.getodk.org/t/odk-central-v1-4/36886>`_